news/2023/2023-12-12-how-advances-in-technology-affected-the-osu-community.md

```diff
@@ -0,0 +1,103 @@
+
+It's time to dive in and see how technology can impact the community, what the results of widespread adoption are, or could have been.
+
+## Graphics tablets (2007)
+
+osu! originally started out as an *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan)* clone for PC, which was a game that was exclusive to the Nintendo DS, a console that came, and could be used with the included stylus pen. When [peppy](https://osu.ppy.sh/users/2) released osu!, it was only natural for players who already owned a graphics tablet to try it out in the so far unheard-of osu! context, and since the get-go it was considered a commodity input device that would more closely replicate the original Nintendo DS experience.
+
+Playing with a tablet is a completely different ball game compared to the mouse. Rather than being a direct upgrade by eliminating mouse drift with its absolute positioning, it brings up new challenges and questions regarding its optimal usage. A player figuring out their ideal pen grip, tablet area, and dragging or hovering play style have all been detrimental to increasing the learning curve, which is why some players choose to remain on the much more familiar and intuitive mouse.
+
+Over the years many top players and regular players started switching over to tablet in order to avoid issues associated with mice. Still, there has always been both mouse and tablet players that would compete at the very top of the leaderboards, as both are considered viable for competitive play, each with its own upsides and downsides.
+
+A great video by [Willy](https://osu.ppy.sh/users/3521482) outlines all of above and more regarding the usage of tablets as input devices for osu!:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/V3K3C94fs1o" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Gaming mechanical keyboards (2010)
+
+Mechanical keyboards have been a thing for decades, however after Razer put [gaming and mechanical switches into a compelling package](https://press.razer.com/product-news/the-worlds-first-mechanical-keyboard-designed-for-gaming-launched-at-gamescom-2010/), that launched the chain reaction that resulted in the mechanical keyboard market we know of today.
+
+By 2010, osu! players were already shifting away from clicking circles with their mouse, to mashing with their Zs and Xs keys, so it would make sense that as these new mechanical keyboards were getting easier and cheaper to adopt into their setups. While there's nothing stopping anyone from playing osu! with the membrane keyboards, the community at large is guiding the newer players to purchase a mechanical keyboard, which is essential to the rhythm-based gameplay.
```

Comment:
```suggestion
By 2010, osu! players were already shifting away from clicking circles with their mouse, to mashing with their Z and X keys, so it would make sense that as these new mechanical keyboards were getting easier and cheaper to adopt into their setups. While there's nothing stopping anyone from playing osu! with membrane keyboards, the community at large is guiding the newer players to purchase a mechanical keyboard, which is essential to the rhythm-based gameplay.
```
plural form is already established by `keys`